README.md

```diff
@@ -28,22 +28,18 @@ You should set up one or both of these projects with the `gcloud`
 command so that you can deploy PR versions and/or manually deploy the
 production version as necessary:
 
-    gcloud config configurations create gotraining
+    gcloud config configurations create gotraining-testing
     gcloud config set account your@email.address
-    gcloud config set project gotraining
-
-Use `gotraining-testing` instead of `gotraining` for the testing
-project.
-
-If you are on the ANZ corp network and are having SSL validation issues
-due to the proxy certs, you can disable SSL validation as a last resort:
-
-    gcloud config set auth/disable_ssl_validation
+    gcloud config set project gotraining-testing
 
 You with need to authenticate the new configuration:
```

Comment:
Something's gone missing here - doesn't look like a sentence.

Reply:
This was the typo I mentioned in channel. s/with/will/ please @anzboi